spmd/tensor/ops/pointwise_ops.py

```diff
@@ -416,3 +419,28 @@ def pointwise_rule(
 
 for op in pointwise_ops:
     DTensor._op_to_rules[op] = pointwise_rule
+
+
+@register_prop_rule("aten._softmax_backward_data.default")
+def softmax_bwd_rule(op_schema: OpSchema) -> OutputSharding:
```

Comment:
I know `softmax_bwd_rule` maybe a pointwise rule, but maybe it's better to categorize the softmax op together? Let's put them together in the math_ops.py, you can call into the pointwise_ops from math_ops.py